src/vllm_tgis_adapter/http.py

```diff
@@ -22,17 +22,26 @@
 logger = init_logger(__name__)
 
 
+def build_http_server(
+    args: argparse.Namespace,
+) -> FastAPI:
+    # builds the vllm api server so we can pass reference to it
+    # within the tgis adapter
+
+    app = build_app(args)
+    return app
```

Comment:
Since we're extracting the app now and making assumptions about its state, let's make this async and run `init_app_state` in here as well